server/trillian_map_server/main.go

```diff
@@ -66,6 +66,7 @@ var (
 	configFile = flag.String("config", "", "Config file containing flags, file contents can be overridden by command line flags")
 
 	useSingleTransaction = flag.Bool("single_transaction", false, "Experimental: use a single transaction when updating the map")
+	largePreload         = flag.Bool("large_preload_fix", true, "Experimental: work-around locking performance issues when using useSingleTransaction mode")
```

Comment:
"useSingleTransaction" -> "single_transaction", needs the flag name not var.

Reply:
Done